### PR TITLE
Added some characters

### PR DESCRIPTION
**Pull request by khush021**
Please review and tell me if there is anything else I can do.

### CODE REVIEW

File: public/filter.json [hunk 1]
```diff
@@ -36,6 +36,10 @@
       "name":"Gandhaari",
       "rename":"Gandhari (Mahabharata)"
     },
+    {
```
Comment (Abhiramborige) [inline public/filter.json:39]: why repeated ?
and also

File: package.json [hunk 2]
```diff
@@ -42,5 +42,17 @@
   },
   "devDependencies": {
     "sass": "^1.42.1"
-  }
+  },
+  "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
```
Comment (Abhiramborige) [inline package.json:46]: dude, could you please remove these added lines.

**Comment by Abhiramborige** [comment]
You have included a lot of characters, thanks for that !

File: package.json (hunk 2 above)
Comment (khush021) [reply package.json:46]: oh yes I am sorry

File: public/filter.json (hunk 1 above)
Comment (khush021) [reply public/filter.json:39]: On it

Comment (khush021) [reply public/filter.json:39]: Done. Please check

File: .env [hunk 3]
```diff
@@ -0,0 +1 @@
+PORT=3000
```
Comment (Abhiramborige) [inline .env:1]: please ignore this file while pushing. please remove it

Comment (khush021) [reply .env]: okay, thank you for telling me that.

**Comment by Abhiramborige** [comment]
thanks for the contribution !

**Pull request merged**